### DISCUSSION TITLE
How do we update the included osquery version in launcher packages created by the manager.

### DISCUSSION THREAD

**Discussion by jtgreen-cse** [Security-Onion-Solutions/securityonion#1578] [Unsupported Versions]
I'd like to start using osquery 4.5.1, but our hosts can't update from the internet. Is there a way to manually update the `so-fleet-launcher` image with newer versions of osquery so it builds packages that includes recent versions?

**Comment by defensivedepth** [comment]
I updated the docs a bit around this issue - https://docs.securityonion.net/en/2.3/osquery.html#agents-updating

Edit - I misread your question. Is there something in particular you need with 4.5.1? I will need to think on this more, but offhand, I don't see an easy way to update those packages without a release on our side / an updated `so-fleet-launcher` image

**Reply by jtgreen-cse** [reply]
Mainly the UTF-8 fixes for Windows events.

**Reply by defensivedepth** [reply]
Ok, I have something that may work... Will let you know.
